### PR TITLE
Move Scheduler related spi util classes into schedulers package

### PR DESCRIPTION
**Pull request by mycFelix**
As #1604 mentioned, this PR is trying to move some `spi-classes` into schedulers package. 

There are five src and two test classes included totally:
```
c.t.h.spi.utils.Runtime
c.t.h.spi.utils.SchedulerConfig
c.t.h.spi.utils.Shutdown
c.t.h.spi.utils.LauncherUtils
c.t.h.spi.utils.SchedulerUtils

c.t.h.spi.utils.SchedulerUtils.LauncherUtilsTest
c.t.h.spi.utils.SchedulerUtils.SchedulerUtilsTest
```

The key changes are following:

* move classes into `c.t.h.schedulers.utils`
* modify `c.t.h.schedulers.BUILD` and modify `c.t.h.scheduler-core.BUILD`
* modify related class invoke info 

I've no idea about how to refactor `SchedulerConfig` class. So this PR only move it into schedulers package without any change. Let me know if there has anything I could do and any comment is welcomed.

cc @billonahill

### CODE REVIEW

File: heron/schedulers/src/java/com/twitter/heron/scheduler/utils/SchedulerConfig.java [hunk 1]
```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package com.twitter.heron.spi.utils;
+package com.twitter.heron.scheduler.utils;
```
Comment (billonahill) [inline heron/schedulers/src/java/com/twitter/heron/scheduler/utils/SchedulerConfig.java]: This class is really a utility, not a config. We should probably name it `SchedulerConfigUtils`. Also all but the last method can be private.

Comment (mycFelix) [inline heron/schedulers/src/java/com/twitter/heron/scheduler/utils/SchedulerConfig.java]: Understood, I'll update it.

File: heron/spi/tests/java/BUILD [hunk 2]
```diff
@@ -42,8 +42,6 @@ java_tests(
     test_classes = [
         "com.twitter.heron.spi.utils.ShellUtilsTest",
         "com.twitter.heron.spi.utils.NetworkUtilsTest",
-        "com.twitter.heron.spi.utils.SchedulerUtilsTest",
```
Comment (mycFelix) [inline heron/spi/tests/java/BUILD]: My bad, forgot doing this.

File: heron/scheduler-core/src/java/BUILD [hunk 3]
```diff
@@ -18,10 +18,15 @@ spi_deps_files = [
     "//heron/spi/src/java:utils-spi-java",
 ]
 
+schedulers_deps_files = [
+    "//heron/schedulers/src/java:utils-schedulers-java",
```
Comment (mycFelix) [inline heron/scheduler-core/src/java/BUILD]: Good Point!  Thank you for your advice. I'll try to move them into schedulers-core package

**Comment by mycFelix** [comment]
Updated

* move classes into `schedulers-core`
* refactor `SchedulerConfig` class
* add `test_classes`

@billonahill Please take a look again when you have a chance.

**Comment by billonahill** [comment]
👍

Thanks for the cleanup!

**Pull request merged**